configure.ac

```diff
@@ -365,6 +365,9 @@ AC_SUBST(PLATFORM_AOT_SUFFIX)
 
 ## PLATFORM_AOT_SUFFIX not so simple for windows :-)
 
+: ${CFLAGS="-g -O3"}
```

Comment:
I get why you did it this way and it is fine, I was just trying to point out that the Jenkins results are not meaningful if you don't modify run-jenkins.sh as well. So yes, please do that :)